lib/url.js

```diff
@@ -92,7 +93,7 @@ function urlParse(url, parseQueryString, slashesDenoteHost) {
 
 Url.prototype.parse = function(url, parseQueryString, slashesDenoteHost) {
   if (typeof url !== 'string') {
-    throw new TypeError('Parameter "url" must be a string, not ' + typeof url);
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE','url','String',url);
```

Comment:
Linter should complain here, there needs to be a space following the comma. Can you run `make lint` to check this?